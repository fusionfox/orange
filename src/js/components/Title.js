import React from 'react'
import PropTypes from 'prop-types'
import '../../css/components/Title.css';

const Title = ({text}) => {
  return (
    <h1 className='Title'>{text}</h1>
  )
}

Title.propTypes = {
  text: PropTypes.string.isRequired
}

export default Title
