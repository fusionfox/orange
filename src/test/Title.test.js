import React from 'react'
import renderer from 'react-test-renderer'
import Title from '../js/components/Title'

test('Text is displayed in a h1 tag', () => {
  const component = renderer.create(
    <Title text="This is a title" />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
});
