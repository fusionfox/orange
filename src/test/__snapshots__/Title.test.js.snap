// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text is displayed in a h1 tag 1`] = `
<h1
  className="Title"
>
  This is a title
</h1>
`;
